Languages/IronPython/Tests/interop/net/derivation/test_property_override.py

```diff
@@ -78,7 +77,10 @@ def get_StrProperty(self):
     x = C()
     p = IProperty11.StrProperty
     p.__set__(x, 'abc')     # no-op equivalent?
-    AssertError(SystemError, lambda: p.SetValue(x, 'def'))  # ?
+    if is_netstandard: # no System.SystemException in netstandard1.6
```

Comment:
Can we add some TODO or something to revert back once we're requiring netstandard2.0?